lib/ansible/modules/apt_repository.py

```diff
@@ -510,7 +522,7 @@ def main():
         argument_spec=dict(
             repo=dict(type='str', required=True),
             state=dict(type='str', default='present', choices=['absent', 'present']),
-            mode=dict(type='raw'),
+            mode=dict(type='raw', default='0644'),
```

Comment:
So I assume `default: '0644'` has to be removed from docs instead of adding it in module argument_spec?

Reply:
@aminvakil yes, imo, a general rule should be like if there are deviations between doc and arg_spec, changing the doc would be preferable because it doesn't affect actual module's behavior, so safer

Reply:
If it would be changed the other way, it would need a `breaking_change` changelog and should better be done by the maintainer(s), since they (hopefully) know what was intended. Maybe in this case we still need a changelog fragment (`bugfixes`?) because some people might have relied on the `0644` assumption and didn't notice it didn't happen.

Reply:
Is this OK?
https://github.com/ansible/ansible/pull/70319/commits/2b346b58ff8b4aa50619220bb3e2515069acd979

Reply:
I think so.